geojson/src/commonTest/kotlin/org/maplibre/spatialk/geojson/FeatureTest.kt

```diff
@@ -298,4 +298,21 @@ class FeatureTest {
             )
         )
     }
+
+    @Test
+    fun testNullGeometry() {
+        val feature =
+            Feature.fromJson<Geometry>(
+                """
+            {
+                "type": "Feature",
+                "geometry": null
+            }
+            """
+            )
+        assertEquals(null, feature.geometry)
+
+        val json = feature.toJson()
+        assertTrue(json.contains("\"geometry\":null"))
```

Comment:
Note that this is a bit unstable, because you are implicitly also testing how the JSON output is formatted (i.e. that there is no space in-between key and value).

```suggestion
        assertEquals(feature, feature.toJson().fromJson<Geometry>())
```

This should be better.